### PR TITLE
US Enrichment tools built. Example implementation not yet created.

### PR DESCRIPTION
**Pull request by ducanbeu**
https://app.clickup.com/t/86aywe0av
https://app.clickup.com/t/86aywdzyn
https://app.clickup.com/t/86aywdzqy

Be sure tests pass/look nice and comprehensive. This infrastructure needs to be merged and versioned before an example can be created.

### CODE REVIEW

File: src/us_enrichment/Client.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
+const Errors = require("../Errors");
+const Request = require("../Request");
+const buildInputData = require("../util/buildInputData");
+const {usEnrichment: keyTranslationFormat} = require("../util/apiToSDKKeyMap");
+const {Response, FinancialResponse, GeoResponse} = require("./Response.js");
```
Comment (smarty-hays) [inline src/us_enrichment/Client.js:5]: Looks like none of these are used when I put it in the IDE... can we delete this?

File: tests/us_enrichment/test_Client.js [hunk 2]
```diff
@@ -0,0 +1,239 @@
+const chai = require("chai");
+const expect = chai.expect;
+const Client = require("../../src/us_enrichment/Client");
+const Lookup = require("../../src/us_enrichment/Lookup");
+const errors = require("../../src/Errors");
+const MockSender = require("../fixtures/mock_senders").MockSender;
+const MockSenderWithResponse = require("../fixtures/mock_senders").MockSenderWithResponse;
+const Attributes = require("../../src/us_enrichment/Response.js")
```
Comment (smarty-hays) [inline tests/us_enrichment/test_Client.js:8]: Looks like this can also be commented out

File: tests/us_enrichment/test_Client.js [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+const {Response, FinancialResponse, GeoResponse} = require("../../src/us_enrichment/Response");
+
+describe("A US Autocomplete Pro Client", function () {
```
Comment (smarty-hays) [inline tests/us_enrichment/test_Client.js:11]: This is super minor, but do we want to change this to be US Enrichment?

File: tests/us_enrichment/test_Client.js (hunk 2 above)
Comment (ducanbeu) [reply tests/us_enrichment/test_Client.js:8]: Line 5 has been removed on the most recent commit.

File: src/us_enrichment/Client.js (hunk 1 above)
Comment (ducanbeu) [reply src/us_enrichment/Client.js]: Line 5 has been removed on the most recent commit.

File: tests/us_enrichment/test_Client.js (hunk 2 above)
Comment (ducanbeu) [reply tests/us_enrichment/test_Client.js]: Removed line 8

File: tests/us_enrichment/test_Client.js (hunk 3 above)
Comment (ducanbeu) [reply tests/us_enrichment/test_Client.js]: Good call. Got that changed

**Review by smarty-hays — APPROVED** [review]

**Pull request merged**